### ISSUE TITLE
Change in abbreviations ?

### ISSUE THREAD

**Issue by loggin** [emmetio/emmet#607]
Using Sublime text with Emmet

This is a probably a question but abbreviations mostly in CSS are not behaving as they have been in the past due to an update in recent months (can't remember exactly when due to being furloughed under the current circumstances) i.e. I used to be able type `d:ib` and tab to get `display:inline-block`, I have discovered I can type `dib` to get the intended expansion but years of habit are little awkward to change and also the [cheatsheet ](https://docs.emmet.io/cheat-sheet/) does not document these changes, is this intentional or is this a bug ?

**Comment by sergeche** [comment]
The same behavior was in previous version as well. The reason is to reduce the amount of extra characters to type that requires Shift key

**Comment by loggin** [comment]
Is there any updated documentation anywhere as I'm not always getting what I want when removing the semicolon ?

**Comment by sergeche** [comment]
No, docs are outdated, maybe some day I can create new version. You can also use - instead of :
